### PR TITLE
Add DB and API methods to get all cards

### PR DESCRIPTION
**Pull request by fjork3**

### CODE REVIEW

**Review by JacobGinsparg — APPROVED** [review]
Wondering if there's a way to combine the tests you added since they're essentially the same, but everything looks good aside from that.

File: src/scooze/api/__init__.py [hunk 1]
```diff
@@ -197,6 +197,16 @@ def get_cards_by_set(self, set_code: str) -> List[CardT]:
...
         )
 
+    @_check_for_safe_context
+    def get_cards_all(self) -> List[CardT]:
+        """
+        Get all cards from the database. WARNING: may be extremely large.
+
+        Returns:
+            A list of all cards in the database.
+        """
```
Comment (iambroadband) [inline src/scooze/api/__init__.py:207]: new line

File: src/scooze/database/core.py [hunk 2]
```diff
@@ -123,6 +123,21 @@ async def get_random_documents(col_type: DbCollection, limit: int):
     return await db.client.scooze[col_type].aggregate(pipeline).to_list(limit)
 
 
+async def get_all_documents(col_type: DbCollection):
```
Comment (iambroadband) [inline src/scooze/database/core.py:126]: nit - can we make a ticket to change all occurences of col_type to coll_type? col feels like column in this context even though I know it's NoSQL

**Comment by iambroadband** [comment]
Looks good. Small formatting nits and one about a variable name that's used elsewhere anyway.

File: src/scooze/database/core.py (hunk 2 above)
Comment (fjork3) [reply src/scooze/database/core.py:127]: done: https://github.com/arcavios/scooze/issues/174

**Review by iambroadband — APPROVED** [review]

**Pull request merged**